src/app/drive/components/EditItemNameDialog/EditItemNameDialog.tsx

```diff
@@ -22,9 +22,10 @@ const EditItemNameDialog: FC<EditItemNameDialogProps> = ({ item, isOpen, resourc
   const [error, setError] = useState('');
   const [isLoading, setIsLoading] = useState(false);
   const [newItemName, setNewItemName] = useState('');
+  const itemName = item?.plainName ?? item?.plain_name ?? '';
```

Comment:
Oops, do we still have any cases where we get the plain name in snake_case? 
Can you check if it is feasible to remove this field using this fix?